web/src/message_list_tooltips.ts

```diff
@@ -262,7 +262,7 @@ export function initialize(): void {
 
     message_list_tooltip(".code_external_link");
 
-    message_list_tooltip(".change_visibility_policy > i", {
+    message_list_tooltip(".change_visibility_policy_tooltip > i", {
```

Comment:
Let's scope this to `#message_feed_container` and drop the commit named `tooltip: Refactor topic visibility policy indicator tooltip.`.